src/app/components/search/enums/error.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum ERROR_MESSAGES {
```

Comment:
Is this used? I can't see any references to it.

Reply:
No it's not used, I will remove this.